### ISSUE TITLE
Release Twisted 16.6.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8897]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8897|
|Type|enhancement|
|Created|2016-11-10 04:30:47Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.6.0-8897|

<details><summary>Searchable metadata</summary>

```
trac-id__8897 8897
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__release_16_6_0_8897 release-16.6.0-8897
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1478752247367903 1478752247367903
changetime__1480103933780807 1480103933780807
version__None None
owner__Amber_Brown__HawkOwl___hawkowl_____ Amber Brown (HawkOwl) <hawkowl@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Prerelease is out

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl removed owner|
|-|-|

pls review

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_4"></a>|@markrwilliams commented|
|-|-|

news is good.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set owner to @hawkowl<br>@hawkowl set status to `closed`|
|-|-|

In changeset d12bb01e44fa0e348244d1ac27012d36a2a1a410
```
#!CommitTicketReference repository="" revision="d12bb01e44fa0e348244d1ac27012d36a2a1a410"
Merge release-16.6.0-8897: Release Twisted 16.6

Author: hawkowl
Reviewer: markrwilliams
Fixes: #8897
```
